infrastructure/main.tf

```diff
@@ -152,51 +152,51 @@ module "db-profile-sync-data" {
   common_tags = "${var.common_tags}"
 }
 
-module "rd_profile_sync" {
-  source = "git@github.com:hmcts/cnp-module-webapp?ref=master"
-  product = "${var.product}-${var.component}"
-  location = "${var.location}"
-  env = "${var.env}"
-  ilbIp = "${var.ilbIp}"
-  resource_group_name = "${azurerm_resource_group.rg.name}"
-  subscription = "${var.subscription}"
-  capacity = "${var.capacity}"
-  instance_size = "${var.instance_size}"
-  common_tags = "${merge(var.common_tags, map("lastUpdated", "${timestamp()}"))}"
-  appinsights_instrumentation_key = "${var.appinsights_instrumentation_key}"
-  asp_name = "${local.app_service_plan}"
-  asp_rg = "${local.app_service_plan}"
-  enable_ase = "${var.enable_ase}"
-
-  app_settings = {
-    LOGBACK_REQUIRE_ALERT_LEVEL = false
-    LOGBACK_REQUIRE_ERROR_CODE = false
-
-    POSTGRES_HOST = "${module.db-profile-sync-data.host_name}"
-    POSTGRES_PORT = "${module.db-profile-sync-data.postgresql_listen_port}"
-    POSTGRES_DATABASE = "${module.db-profile-sync-data.postgresql_database}"
-    POSTGRES_USER = "${module.db-profile-sync-data.user_name}"
-    POSTGRES_USERNAME = "${module.db-profile-sync-data.user_name}"
-    POSTGRES_PASSWORD = "${module.db-profile-sync-data.postgresql_password}"
-    POSTGRES_CONNECTION_OPTIONS = "?"
-
-    S2S_URL = "${data.azurerm_key_vault_secret.s2s_url.value}"
-    S2S_SECRET = "${data.azurerm_key_vault_secret.s2s_secret.value}"
-    IDAM_URL = "${data.azurerm_key_vault_secret.idam_url.value}"
-    USER_PROFILE_URL = "${data.azurerm_key_vault_secret.USER_PROFILE_URL.value}"
+# module "rd_profile_sync" {
```

Comment:
Can we not remove commented code as old code will be still in git history?

Reply:
Yes, this can be removed. Note that, Terraform will delete the Web App resource when the demo build runs. Please let me know if this needs to be retained, until AKS migration is successful.